docs/docs/how-tos/configure-keycloak-howto.md

```diff
@@ -0,0 +1,167 @@
+---
+id: configuring-keycloak
+title: Configure Keycloak
+description: |
+  How to configure Keycloak, add new users, add new groups and update roles.
+---
+
+## Introduction to Keycloak
+
+[Keycloak](https://www.keycloak.org/) is the open-source identity and access management tool that comes standard with Nebari. It is used as a centralized location to add new users, create user groups and update roles and permissions. Keycloak can be configured to work with your existing identity provider services such as GitHub, Auth0 and many others.
+
+:::note
+Although Nebari comes hooks to connect with GitHub and Auth0 from the `nebari-config.yaml`, any identity provider that Keycloak can connect to will work. See these [Keycloak docs](https://www.keycloak.org/docs/latest/server_admin/#_identity_broker) for more information.
```

Comment:
```suggestion
Although Nebari allows you to connect with GitHub and Auth0 from the `nebari-config.yaml`, any identity provider that Keycloak can connect to will work. See these [Keycloak docs](https://www.keycloak.org/docs/latest/server_admin/#_identity_broker) for more information.
```

I think this is what we're trying to convey, but please update as you feel is accurate!